templates/cassiopeia/index.php

```diff
@@ -41,9 +41,14 @@
 // Load custom Javascript file
 HTMLHelper::_('script', 'user.js', ['version' => 'auto', 'relative' => true]);
 
+//HTMLHelper::_('script', 'media/vendor/responsifyjs/responsify.js', ['relative' => false]);
+
 // Load template CSS file
 HTMLHelper::_('stylesheet', 'template.css', ['version' => 'auto', 'relative' => true]);
 
+HTMLHelper::_('script', 'media/vendor/responsifyjs/responsify.js', ['version' => 'auto', 'relative' => false]);
+
+
```

Comment:
Same as above

Reply:
@dgrammatiko Thank you for review, the reason I have done like this is they are jQuery dependent, So when I tried to load these through plugin they load at first, before any other script, and pop out errors in the console.